workflow-bot-app/src/main/java/com/symphony/bdk/workflow/engine/executor/request/ExecuteRequestExecutor.java

```diff
@@ -45,6 +52,30 @@ public void execute(ActivityExecutorContext<ExecuteRequest> execution) throws IO
     execution.setOutputVariables(outputs);
   }
 
+  private String encodeQueryParameters(String fullUrl) {
```

Comment:
we may want to extract this and unit test it separately to cover different use cases